upload_specifications/readme.md

```diff
@@ -0,0 +1,2 @@
+This directory contains templates for bulk upload specifications to be parsed by the
+upload specfication endpoint, and example code for how to generate them automatically.
```

Comment:
I didn't catch anything missing for the templates

Reply:
Files LGTM.